ios/ElectrodeReactNativeBridge.xcodeproj/project.pbxproj

```diff
@@ -1126,4 +1139,4 @@
 /* End XCConfigurationList section */
 	};
 	rootObject = DA4BE0B31E36DBCD00DBA1ED /* Project object */;
-}
+}
```

Comment:
If this was automatically removed, can you please manually add the now-missing newline at EOF?